hal/src/photon/wiced/RTOS/NuttX/ver7.8/configs/freedom-kl25z/include/board.h

```diff
@@ -218,12 +218,6 @@
 #define PIN_SPI1_MISO  (PIN_SPI1_MISO_3 | PIN_ALT2_PULLUP)
 #define PIN_SPI1_MOSI  (PIN_SPI0_MOSI_7 | PIN_ALT2_PULLUP)
 
-/* These pins are used by CC3000 module */
-
-#define GPIO_WIFI_EN (GPIO_OUTPUT | GPIO_OUTPUT_ZER0 | PIN_PORTC | PIN12)
```

Comment:
This is not part of the DeviceOS and is a header coming from WICED (which is bundled with DeviceOS). It also doesn't talk about the Core, it's just CC3000 (TI wifi chip) support for some platform that WICED supports.

Change should be reverted.

Reply:
Good catch! I thought I had a filter to protect against these directories.